### ISSUE TITLE
migration module: add ability to copy to pool group w/o specifying the target

### ISSUE THREAD

**Issue by DmitryLitvintsev** [dCache/dcache#7623]
Many a times a pool needs to be drained to be decomissioned. Typically the replicas need to go to the same pool group this pool belongs to. Currently you have to explicitly specify:

```
... -target=pgroup PoolGroupName
```

It would be nice just drop this extra parameters and by default 

```
migration copy
```

would copy replicas to pool group(s) the pool belongs to. 

NB: pool may belong to multiple pool groups.  Mabe alpha version of this functionality should refuse migration (fail) if pool belongs to multiple pool groups with clear message.

**Comment by DmitryLitvintsev** [comment]
The behavior has to be backward compatible on the folllowin way:

```
migration copy ..... foo
```
Assumes, as before, that `foo` is destination pool

Whereas :

```
migration copy ... -target=pgroup
```
without specifying pool group name triggers this migration into the same pool group

invoked w/o `-target` and without pool name arguments

```
migration copy ... 
```

has to fail just like it does now (this is somethig I have changed my mind about since Tuesday). I think this is safer as provides exact same behavior as before.

**Comment by greenc-FNAL** [comment]
Implemented via [cfeed0cc3ce399056472b5361b19dee433b0de4a](https://github.com/dCache/dcache/commit/cfeed0cc3ce399056472b5361b19dee433b0de4a)
